openquake/hazardlib/shakemap/parsers.py

```diff
@@ -60,6 +60,7 @@
     'STDPSA10': ('std', 'SA(1.0)'),
     'STDPSA30': ('std', 'SA(3.0)'),
 }
+REQUIRED_IMTS = {'PGA', 'PSA03', 'PSA10'}
```

Comment:
Not sure if this should be enforced while parsing the ShakeMap grid files. An IMTs is "required" only if the subsequent risk calculation uses fragility/vulnerability/consequence functions that use that IMT.

Reply:
Unfortunately the current implementation (done according to Vitor's specs) really requires PGA, SA(0.3) and SA(1.0) in the sense that they are stored inside gmf_data, even if the calculation does not require such IMTs (before they were stored with zero values, now there would be an error). Changing the current implementation is outside the scope of this PR.